server/middleware/isAdmin.js

```diff
@@ -0,0 +1,19 @@
+const { checkAdminQuery } = require('../database/quieres');
+
+const isAdmin = async (req, res, next) => {
+  try {
+    const { email } = req.cookies;
```

Comment:
This should be a token and then we have to decode it and check that the email in the token is really an admin and since there's no role here we have to look through the database, but maybe in the future when you create the login for the admin you would add rule and leave the looking through the database